wled00/FX.cpp

```diff
@@ -5021,10 +5024,11 @@ uint16_t mode_2DDNASpiral() {               // By: ldirko  https://editor.soulma
       SEGMENT.setPixelColorXY(x1, i, WHITE);
     }
   }
+  SEGMENT.blur(SEGMENT.custom1 >> (!SEGMENT.check1), SEGMENT.check1); 
```

Comment:
This will cause *different looking* effect on existing presets. Please consider backwards compatibility.

Reply:
in FX where I added blur, the default slider position is set to 0, checks are set to 0.
that will change behaviour on users that have 'default fx parameters' unused. if that is a restriction, none of the FX can be extended (ever).